### PR TITLE
Add config for IGMP snooping on BDN

### PR DESCRIPTION
**Pull request by oneswig**

### CODE REVIEW

File: etc/kayobe/inventory/host_vars/ethsw-b17-u40-41 [hunk 1]
```diff
@@ -60,19 +60,26 @@ switch_interface_config:
   Te1/1/3/4:
     config: "{{ switch_interface_config_all }}"
   Hu1/4/7:
-    description: ethsw-b16-u40-41-trunk1
+    description: ethsw-b17-u40-41-trunk1
```
Comment (markgoddard) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: I think these should stay as b16 - they are lag members to b16.

Comment (oneswig) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: In which case should we rename the ports at the other end of the lag?  Both are named the same.

Comment (markgoddard) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: Ah, you're right. The descriptions in b16-u40-41 should be changed.

**Comment by markgoddard** [comment]
What happened to this?

File: etc/kayobe/inventory/host_vars/ethsw-b17-u40-41 (hunk 1 above)
Comment (markgoddard) [inline etc/kayobe/inventory/host_vars/ethsw-b17-u40-41]: I still think this (and the others below) needs to go back to b16.

**Pull request closed without merging**